espnet2/bin/enh_inference.py

```diff
@@ -0,0 +1,194 @@
+#!/usr/bin/env python3
+import logging
+import sys
+from typing import Optional
+from typing import Sequence
+from typing import Tuple
+from typing import Union
+
+import configargparse
+import torch
+from typeguard import check_argument_types
+
+from espnet.utils.cli_utils import get_commandline_args
+from espnet2.fileio.sound_scp import SoundScpWriter
+from espnet2.tasks.enh import EnhancementTask
+
```

Comment:
```suggestion
```

Please also care about the importing order.

```
<officiai python module>

<third party>

<internal modules>
```

Reply:
Done.